examples/catalog/lib/custom_a11y_traversal.dart

```diff
@@ -0,0 +1,342 @@
+                'How many pets do you own?',
+                textScaleFactor: 2.0,
+              ),
+            ),
+            const Padding(padding: const EdgeInsets.symmetric(vertical: 10.0)),
+            new Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                _makeFieldHeader(1, 0, Field.DOGS),
+                _makeFieldHeader(1, 1, Field.CATS),
+                _makeFieldHeader(1, 2, Field.FISH),
+              ],
+            ),
+            new Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                _makeSpinnerButton(3, 0, Field.DOGS, increment: true),
+                _makeSpinnerButton(3, 1, Field.CATS, increment: true),
+                _makeSpinnerButton(3, 2, Field.FISH, increment: true),
+              ],
+            ),
+            new Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                _makeEntryField(2, 0, Field.DOGS),
+                _makeEntryField(2, 1, Field.CATS),
+                _makeEntryField(2, 2, Field.FISH),
+              ],
+            ),
+            new Row(
+              mainAxisAlignment: MainAxisAlignment.spaceAround,
+              children: <Widget>[
+                _makeSpinnerButton(4, 0, Field.DOGS, increment: false),
+                _makeSpinnerButton(4, 1, Field.CATS, increment: false),
+                _makeSpinnerButton(4, 2, Field.FISH, increment: false),
+              ],
+            ),
+            const Padding(padding: const EdgeInsets.symmetric(vertical: 10.0)),
+            new Semantics(
+              // Since this is the only sort key that the reset button has, it
+              // will be compared with the 'column' OrdinalSortKeys of all the
+              // fields, because the column sort keys are first in the other fields.
+              //
+              // an ordinal of "5.0" means that it will be traversed after column 4.
+              sortKey: const OrdinalSortKey(5.0),
+              child: new MaterialButton(
+                child: const Text('RESET', textScaleFactor: 2.0),
+                textTheme: ButtonTextTheme.normal,
+                textColor: Colors.blue,
+                onPressed: () {
+                  setState(() {
+                    fields = <int>[0, 0, 0];
+                  });
+                },
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+void main() {
+  runApp(new CustomTraversalExample());
+}
+
+/*
+Sample Catalog
+
+Title: CustomTraversalExample
+
+Summary: An app that demonstrates a custom semantics traversal order.
+
+Description:
+This app presents a value selection interface where the fields can be
+incremented or decremented using spinner arrows. In accessibility mode, the
+widgets are traversed from one column to the next, starting with the column title, moving to
```

Comment:
... are traversed *in custom order* from ...

Reply:
Done.